pkg/kube/krt/index.go

```diff
@@ -17,6 +17,7 @@ package krt
 import (
 	"fmt"
 
+	"github.com/google/uuid"
```

Comment:
Do we need a uuid? We just require some unique ID within the scope of a single running istiod. It doesn't need to be universal as far as I can see so it seems like we could use `collectionUID` from `pkg/kube/krt/internal.go` or similar. Maybe if don't like it because the name we could rename to a more generic krtUID.

Reply:
:+1: